### DISCUSSION TITLE
Please help explain more about the basic conceptions.

### DISCUSSION THREAD

**Discussion by nobody4t** [milvus-io/milvus#27997] [Q&A and General discussion]
![multi-tenancy](https://github.com/milvus-io/milvus/assets/2303377/25f5b730-e6ac-46ed-8e0f-c3783ed1d6a9)

This table is from doc here: https://milvus.io/docs/multi_tenancy.md

But I did not find any explanation for each of them.
With RBAC enabled, user can be assigned collections. So I conclude that multi-tenancy can be achieved along with RBAC, right?

If I use partion based solution, I can assign the partition to users too, because partitons is iniclueded into collections. Now each collection will include only one partition, so that I can assign it to a user. Is this right?

what about partition-key based? How to use RBAC to control its access?
From the doc, it says if using partition-key based solution, the max should be no limit. But i can not figure this out. Why?
The number of partition-key is no-limit? But from the doc, only one partition-key permitted for the collection, right? So it got limits. Please explain this.

Furthermore:
The max partitions in a collection is 4096. And the max partitions in collection is also 4096. What is the relationship bewteen them? I can not figure it out.

From the docs, the limit of collection partitions is 4096, if I use them to store different users' data, then all the partitions I can. get are num of collections multiplied by 4096, it should be 65532*4096. But the max partition of a instance is 4096. I am confusing.

**Comment by nobody4t** [comment]
@xiaofan-luan Would you please help demysify this? Thanks.

**Comment by xiaofan-luan** [comment]
RBAC is on collection level, while partition key is in one partition.

Partition key can be any size, it could be up to 10 million partition keys or more.

Each partition key does not map to a partition. partition key is a logical concept but partition is a physical concept

**Comment by nobody4t** [comment]
@xiaofan-luan Thanks for your response.

>Each partition key does not map to a partition. partition key is a logical concept but partition is a physical concept

if  a parttion key is setup when to create collection, then all the data will be distributed according to the value of its key, to different partitions? Am I right here? The partitions will be re-used ?
I can not get it here. Would you please help explain in more details about it?

And each collection can have 4096 partitions, and why the partitions of a milvus is 4096 not 65536*4096?
